x-pack/plugins/observability/public/components/app/section/ux/index.tsx

```diff
@@ -81,3 +115,10 @@ export function UXSection({ bucketSize }: Props) {
     </SectionContainer>
   );
 }
+
+const PAGE_LOAD_DISTRIBUTION_TITLE = i18n.translate(
+  'xpack.observability.overview.ux.pageLoadDistribution.title',
+  {
+    defaultMessage: 'Page load distribution',
```

Comment:
```suggestion
    defaultMessage: 'Page load distribution: Top 5 services',
```

Not sure if we're scoping it to the top 5 services instead of all services available? If so, we can add that to the title of the chart.

Reply:
Good point, I totally forgot about it 😅. I've been searching for how to do it in the Exploratory view but I didn't know how, so maybe we can't do that.

@shahzad31 is it possible to show the top 5 services only in the exploratory view?

Reply:
@estermv right now, it's hardcode, so we will have to make it configurable, if we want that. 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts#L174

Reply:
i will suggest creating an enhancement request

Reply:
@formgeist I opened an issue to make it configurable https://github.com/elastic/kibana/issues/132720. Since the number of services is already limited and the chart won't break, are you fine with merging this as it is?